reference/gdscript.rst

```diff
@@ -799,9 +799,12 @@ special export syntax is provided.
     export(int) var number
 
     # Export can also take a resource type to use as a hint
-
+    
+    # Select Texture image
     export(Texture) var character_face
-
+    # Select a scene file from your project
+    export (PackedScene) var scene_file
+   
```

Comment:
I don't see how that brings any consistency?
![image](https://cloud.githubusercontent.com/assets/4701338/19727138/7809b58e-9b8e-11e6-89ff-900e5f888029.png)